packages/storybook-addon/src/decorator/preact.ts

```diff
@@ -1,18 +1,22 @@
 /** FUTURE EXPORT - NOT CURRENTLY BEING USED */
 import { addDecorator } from '@storybook/preact';
-import { Provider, createClient } from '@urql/preact';
+import { Provider, createClient, ClientOptions } from '@urql/preact';
 import { devtoolsExchange } from '@urql/devtools';
 import { createElement } from 'preact';
 import { getStorybookExchange } from '../exchange';
 
-export const urqlDecorator: Parameters<typeof addDecorator>[0] = (
-  Story,
-  context
-) => {
+const createDecorator = (
+  opts?: Partial<ClientOptions>
+): Parameters<typeof addDecorator>[0] => (Story, context) => {
   const client = createClient({
-    url: '/graphql',
+    ...opts,
+    url: 'storehhh',
```

Comment:
ehhh, any reason we're setting it to this value in particular? 😄

Reply:
It's more representative of what folks are going to be using in the real world.

JK - no idea how that slipped in